policyengine_us/parameters/gov/states/ri/tax/income/adjusted_gross_income/subtractions/child_tax_rebates/max_child.yaml

```diff
@@ -0,0 +1,16 @@
+description: Rhode Island limits the maximum number of children eligible under the child tax rebate.
+
+metadata:
+  unit: person
+  label: Rhode Island child tax rebate child cap
+  period: year
+  reference:
+  #Rhode Island started issuing child tax rebates on 2022
+  - title: 2022 RI Schedule M - page 1 RI Modifications to Federal AGI line 1v
+    href: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20RI%20Schedule%20M_w.pdf#page=1
```

Comment:
this file does not show the number "3" - check file references in other yaml files as well

Reply:
Hi Pavel! This pdf points that we need to add Child Tax Rebate in the subtractions, and the legal code below shows the detailed steps which include the maximum of 3 children. Should we delete the first one and only keep the legal code reference?

Reply:
yes, and we need to see if the child cap is mentioned in the tax form and reference the right page

Reply:
Yes. The file below shows " up to a maximum of three children" in (b), (1)